### PR TITLE
names: renaming 'extension' things to be 'stack' things, updating stack to assume default policies

### PR DESCRIPTION
**Pull request by suskin**
Related to crossplaneio/crossplane#732

## Overview

We renamed crossplane extensions to be crossplane stacks, including
stack structure folder names and Kinds, so this renames them in this
stack too.

## Testing done

Ad-hoc local testing by building and installing the extension, against crossplaneio/crossplane@d9783258f6538db78c9163e6860ee2100db9ca16

## Notes for reviewers

This is more intended as an FYI than a thing that will block on a review before being merged, but feel free to leave comments! I'll revisit any comments in a future changeset.

### CODE REVIEW

File: api/v1alpha1/wordpressinstance_types.go [hunk 1]
```diff
@@ -56,7 +56,7 @@ type WordpressInstanceList struct {
 	Items           []WordpressInstance `json:"items"`
 }
 
-// ExtensionRequest type metadata.
+// StackRequest type metadata.
```
Comment (jbw976) [inline api/v1alpha1/wordpressinstance_types.go]: this appears to be for the WordpressInstance type, not for extensions or stacks

Comment (suskin) [reply api/v1alpha1/wordpressinstance_types.go]: good catch! o:

**Comment by suskin** [comment]
The reason I have not merged this changeset yet is because during testing I discovered that there have been other changes that we should update the wordpress stack to pick up. The changes are around claims: strongly typed claims have landed, and default policies are the mechanism used to specify a portable workload now. I'm currently working on some things on the Stack CLI, but will return to this changeset at some point and roll those other changes into it, unless someone else gets to it first.

**Comment by suskin** [comment]
> The reason I have not merged this changeset yet is because during testing I discovered that there have been other changes that we should update the wordpress stack to pick up

The most recent commits address these updates, and have been tested locally.

File: api/v1alpha1/wordpressinstance_types.go (hunk 1 above)
Comment (suskin) [reply api/v1alpha1/wordpressinstance_types.go]: Fixed

**Pull request merged**